draft-ietf-mls-protocol.md

```diff
@@ -2368,8 +2373,14 @@ If there are multiple proposals that apply to the same leaf, the committer
 chooses one and includes only that one in the Commit, considering the rest
 invalid. The committer MUST prefer any Remove received, or the most recent
 Update for the leaf if there are no Removes. If there are multiple Add proposals
-for the same client, the committer again chooses one to include and considers
-the rest invalid.
+referencing the same `key_package`, the committer again chooses one to include
+and considers the rest invalid.
+
+If there are multiple Add proposals with KeyPackages belonging to the same
+identity, the committer SHOULD choose one and consider the rest invalid.
+Similarly, if there are Add proposals including KeyPackages belonging to an
+identity that is already present as a member of the group, the committer SHOULD
```

Comment:
"if there are Add proposals of KeyPackages with an identity that is already present"